kolibri/core/logger/csv_export.py

```diff
@@ -103,7 +124,37 @@ def cache_content_title(obj):
     )
 )
 
-map_object = partial(output_mapper, labels=labels, output_mappings=mappings)
+
+def get_max_ancestor_depth():
+    max_depth = 0
+    content_ids = ContentSummaryLog.objects.values_list("content_id", flat=True)
+    nodes = (
+        ContentNode.objects.filter(content_id__in=content_ids)
+        .only("content_id")
```

Comment:
Any reason not to get the title and ancestors fields in this fetch and seed the cache with it?

That would bring us from N + 1 queries, down to N / BATCH_SIZE queries!